### ISSUE TITLE
TBCLeaRingSlider ring alignment

### ISSUE THREAD

**Issue by circular17** [bgrabitmap/bgracontrols#207]
The ring is not properly aligned with the disk when the component is not a square (when width/height differs):

![image](https://github.com/user-attachments/assets/f893667c-8b8e-43f7-b6e8-0f5f9e206fd5)

When aligned it looks like this:

![image](https://github.com/user-attachments/assets/2ffd275e-5ca4-4542-87fc-c79c057e845c)

**Comment by circular17** [comment]
There is the same problem with TLeaSelector.

**Comment by maxm74** [comment]
I noticed the same thing when I added text alignment. Ideally if you want a circle you should keep the square shape when you resize, otherwise you should use an ellipse.

**Comment by circular17** [comment]
Yes, I would fine the ellipse fine as well and in fact provide more possibilities.

**Comment by lainz** [comment]
melchiorrecaruso fixed it for TBCLeaRingSlider, missing TLeaSelector.
